include/constants/flags.h

```diff
@@ -1573,7 +1573,7 @@
 #define FLAG_UNUSED_0x91F                           (SYSTEM_FLAGS + 0xBF) // Unused Flag
 
 // Daily Flags
-#define DAILY_FLAGS_START                           0x920
+#define DAILY_FLAGS_START                           (FLAG_UNUSED_0x91F + 1)
```

Comment:
Maybe make this relative to SYSTEM_FLAGS, and/or force alignment? idk whether the daily flags are reset by clearing that part of the array directly

Reply:
Yeah they're cleared directly (`ClearDailyFlags`). I'll force alignment, otherwise itll round down and potentially clear some system flags